bedrock/firefox/templates/firefox/welcome/page8.html

```diff
@@ -14,20 +14,71 @@
 {% set _utm_campaign = 'welcome-8-protection' %}
 
 {% block content_intro %}
+
   {% call hero(
       title=ftl('welcome-page8-your-privacy-respected'),
       desc=ftl('welcome-page8-firefox-automatically-protects'),
-      class='mzp-t-firefox mzp-t-dark',
+      class='mzp-t-firefox mzp-t-dark variation-default',
       include_cta=True,
       heading_level=1
     ) %}
+    {% endcall %}
+    <section class="mzp-c-hero mzp-t-firefox mzp-t-dark" id="hero-image">
+      <div class="mzp-l-content">
+        <div class="mzp-c-hero-body">
+          <h1 class="mzp-c-hero-title">{{ ftl('welcome-page8-your-privacy-respected') }}</h1>
+          <div class="image-container">
+            <div>
+              <img src="{{ static('img/firefox/welcome/trackers-magnify.svg') }}" alt="stop trackers from watching you" />
+              <p>{{ ftl('welcome-page8-tracker-watching-description') }}</p>
+            </div>
+            <div>
+              <img src="{{ static('img/firefox/welcome/trackers-money.svg') }}" alt="stop trackers from selling your data" />
+              <p>{{ ftl('welcome-page8-tracker-money-description') }}</p>
+            </div>
+            <div>
+              <img src="{{ static('img/firefox/welcome/trackers-shield.svg') }}" alt="auto-block trackers by default" />
+              <p>{{ ftl('welcome-page8-tracker-auto-blocking-description') }}</p>
+            </div>
+          </div>
+
+        <p class="primary-cta protection-report">
+          <a href="https://support.mozilla.org/kb/enhanced-tracking-protection-firefox-desktop/?src=external-www.mozilla.org-welcom-7&amp;utm_source=mozilla.org-firefox-welcome-8&amp;utm_medium=referral&amp;utm_campaign=welcome-8-protection" class="mzp-c-button mzp-t-product" id="button" rel="external">
+            {{ ftl('welcome-page8-view-your-protection-report') }}
+          </a>
+        </p>
+        </div>
+      </div>
+      
+    </section>
+
+    <section class="mzp-c-hero mzp-t-firefox mzp-t-dark" id="hero-animation">
+      <div class="mzp-l-content">
+        <div class="mzp-c-hero-body">
+          <h1 class="mzp-c-hero-title">{{ ftl('welcome-page8-your-privacy-respected') }}</h1>
+          <div class="image-container">
+            <video autoplay loop playsinline muted>
+              <source src="{{ static('img/firefox/welcome/Animation@1x.mp4') }}" type="video/mp4">
```

Comment:
I'm not sure if the content of this video is final, but it currently contains a fair amount of English copy, which is fine for an experiment but may not work well for many non-English languages.

Also flagging this as a potential hosting issue depending on the size of the final video file. Our team may need to get back to you on the best place to host this from.

Reply:
I should receive a fresh video today. It will not have any copy in it, only the moving animation parts. I'd appreciate advice on where to store it.

Reply:
Can you please let me know the size of the final video? I’ll speak to our SRE’s on their preferred place to host this.

Reply:
The final video size is 227kb, and a 2x one at 467kb

Reply:
Ok, we can host this for you on https://assets.mozilla.net/. Please let me know when you have the final video file and we can get it uploaded. Thanks

Reply:
video file hosted here: https://assets.moz.works/video/Animation%401x.mp4

Reply:
Ok here's a link to the current video file: https://assets.moz.works/video/Animation%401x.mp4

We can update / replace this when you have the final assets. Also, the file might be smaller as a webm maybe?

Reply:
Actually, you may need to use https://assets.mozilla.net/video/Animation%401x.mp4 (since *.mozilla.net is in our current CSP rules, whereas *moz.works is not.

Reply:
It looks like you have the old video hosted there, I will send you the new one

Reply:
@ericawright sent me the following files that have been uploaded to our CDN:
https://assets.mozilla.net/video/animation.mp4
https://assets.mozilla.net/video/animation%402x.mp4

Reply:
thank you!